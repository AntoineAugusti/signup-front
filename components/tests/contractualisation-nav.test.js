import ReactTestRenderer from 'react-test-renderer'
import ContractualisationNav from '../contractualisation-nav'

require('../../lib/tests/local-storage') // eslint-disable-line import/no-unassigned-import

describe('components | ContractualisationNav', () => {
  describe('render', () => {
    const props = {
      children: '<p>Some html</p>'
    }
    const renderer = ReactTestRenderer.create(<ContractualisationNav {...props} />)
    it('should be defined', () => {
      expect(renderer).toBeDefined()
    })
    it('should match snapshot', () => {
      expect(renderer).toMatchSnapshot()
    })
  })
})
