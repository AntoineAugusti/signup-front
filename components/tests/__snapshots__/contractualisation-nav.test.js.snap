// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components | ContractualisationNav render should match snapshot 1`] = `
<nav
  className="side-pane"
>
  <ul
    className="side-pane__menu"
  >
    <li
      className="side-pane__title"
    >
      <h3>
        Lorem Ipsum
      </h3>
    </li>
    <li>
      <a
        className="side-pane__link"
        href="#legal"
      >
        Fondement légal
      </a>
    </li>
    <li>
      <a
        className="side-pane__link"
        href="#donnees"
      >
        Choix des données
      </a>
    </li>
    <li>
      <a
        className="side-pane__link"
        href="#volumetrie"
      >
        Volumétrie
      </a>
    </li>
    <li>
      <a
        className="side-pane__link"
        href="#securite"
      >
        Engagement de sécurité
      </a>
    </li>
    <li>
      <a
        className="side-pane__link"
        href="#homologation"
      >
        Homologation
      </a>
    </li>
    <li>
      <a
        className="side-pane__link"
        href="#contrat"
      >
        Documents contractuels
      </a>
    </li>
    <li>
      <a
        className="side-pane__link"
        href="#cnil"
      >
        Obligation CNIL
      </a>
    </li>
    <li>
      <a
        className="side-pane__link"
        href="#recette"
      >
        Recette fonctionnelle
      </a>
    </li>
    <li>
      <a
        className="side-pane__link"
        href="#production"
      >
        Entrants techniques de production
      </a>
    </li>
  </ul>
</nav>
`;
